TABLES_ADD_20240515/sql.js

```diff
@@ -0,0 +1,9 @@
+const CsvSql = require("sql-csv");
+
+const csvSql = new CsvSql("ISDP LOGBOOK REPORT.csv");
+
+async function main() {
+  console.log(await csvSql.query('SELECT * FROM "ISDP LOGBOOK REPORT"'));
+}
+
+main();
```

Comment:
Document each new file you create. Use VSCode to do this. In Python code, please add type hints and docs for module and every function. VSCode+Copilot does that well. Use Problems tab in terminal in VSCode to correct as many as you can. I have the extensions: pylint and flake and pylance. Gives good coverage. I also have the Black formatted extension. When I save a file with cmd-s, the file is automatically formatted. Use defaults for now. We might change that if there are inconsistencies between us.

Reply:
I will add a `pylintrc` file in the project to make it more consistent.

Reply:
Ok. Never used this. Where did you learn about all this?

Reply:
In my previous job, I was developing with TypeScript.